python/langsmith/schemas.py

```diff
@@ -163,6 +158,12 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         """Initialize a Dataset object."""
+        if "inputs_schema_definition" in kwargs:
```

Comment:
this is disgusting

Reply:
Would this actually be applied?

Reply:
wdym? yeah, there's a new integration test showing this works

Reply:
So this is for the dataset copying case ya?

I think it's fine. I also wouldn't resist if we just marked it as "inputs_schema_definition" itself here